pages/docs/reference/delegated-properties.md

```diff
@@ -444,4 +487,21 @@ class C {
 
 </div>
 
-Note that the `provideDelegate` method affects only the creation of the auxiliary property and doesn't affect the code generated for getter or setter.
+Note that the `provideDelegate` method affects only the creation of the auxiliary property and doesn't affect the code
+generated for getter or setter.
+
+With the `PropertyDelegateProvider` interface from the standard library, you can create delegate providers as anonymous
+objects without creating new classes.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only auto-indent="false">
+
+```kotlin
+fun makeDelegate() = PropertyDelegateProvider<Nothing?, Int> =
+    object : PropertyDelegateProvider<Nothing?, Int> {
```

Comment:
Makes sense to showcase SAM constructor syntax being available due to the fact that `PropertyDelegateProider ` is a fun interface

Reply:
See this example in the test related to `PropertyDelegateProvider`: https://github.com/JetBrains/kotlin/blob/de6154980d3f16ccf2bf0fa194ae74802dd51dbc/libraries/stdlib/test/properties/delegation/DelegationTest.kt#L71